Mostly code clean-up's.

eduperez opened the pull request.

0xAF commented:
Edu, I liked more the idea of holding the legacy stuff in a folder. But for the cleaning of the code, it's probably better in separate branch. Most of this legacy code is for reference when you need it. When it is in a separate branch it could be forgotten and never serve as a reference. But since you are the only dev in the project, I guess you know where to look for it ;)
Just one more thing, please add the "legacy" branch to 400plus repo, since it is only in your repo now.

eduperez commented:
The legacy code is still relevant, and a very good reference to me; and precisely because I do not want to lose it that I moved it to a separate branch.

The problem with having the legacy code in a folder is that I am too clumsy, and stumble with it quite frequently... I have modified it several times during a global search-and-replace, and have had to undo the changes, or have searched for users of a function, and have been confused with the legacy code. If I could keep it in a "locked" folder, I would.

At first I preferred to keep only release branches on the 400plus repo, and keep all other branches on the developers' repos; but since the legacy code is not being developed by anyone, I agree that it should be on 400plus' repo. I will move it tonight (and hope not to mess it too much, I still have not mastered Git / GitHub).
